File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int len = s.size(), cnt = 0;
        while(s[len - 1] == ' '){
            len--;
        }
        while(len--){
            if(s[len] == ' ') return cnt;
            else cnt++;
        }
        return cnt;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int len = s.length(), cnt = 0;
        while(s.charAt(len - 1) == ' '){
            len--;
        }
        len--;
        while(len >= 0){
            if(s.charAt(len) == ' ') return cnt;
            else cnt++;
            len--;
        }
        return cnt;
    }
}